bigtop-manager-server/src/main/resources/stacks/bigtop/3.3.0/services/solr/metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ "License"); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~    https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+-->
+<metainfo>
+    <schema-version>2.0</schema-version>
+    <service>
+            <name>solr</name>
+            <display-name>Solr</display-name>
+            <desc>
+                Solr is the popular, blazing-fast, open source enterprise search platform built on Apache Lucene.
+            </desc>
+            <version>8.11.2</version>
+            <user>solr</user>
+            <group>solr</group>
+            <components>
+                <component>
+                    <name>SOLR</name>
+                    <display-name>Solr Instance</display-name>
+                    <category>MASTER</category>
+                    <cardinality>1+</cardinality>
+                    <command-script>
+                        <script-id>org.apache.bigtop.manager.stack.bigtop.v3_3_0.solr.SolrServerScript</script-id>
+                        <script-type>java</script-type>
+                        <timeout>1200</timeout>
+                    </command-script>
```

Comment:
Could you add quicklinks here? you can refers to [hdfs](https://github.com/apache/bigtop-manager/blob/main/bigtop-manager-server/src/main/resources/stacks/bigtop/3.3.0/services/hdfs/metainfo.xml#L58) config